internal/pkg/cli/deploy/workload.go

```diff
@@ -262,12 +270,16 @@ func (d *workloadDeployer) generateCloudFormationTemplate(conf stackSerializer)
 	if err != nil {
 		return nil, fmt.Errorf("generate stack template: %w", err)
 	}
+	transformedTpl, err := d.overrider.Override([]byte(tpl))
```

Comment:
Just curious why does `Override` take and return `[]byte` instead of `string`?

Reply:
I was following the stdlib that usually takes `[]byte` but you're right that at least today we don't expect to override anything that's not a string 😂 .

I'll update the `override` pkg and interfaces in a separate PR to `Override(string) (string, error)` if that sounds ok?